### PR TITLE
ref: add pytest-xdist to development requirements

### PR DESCRIPTION
**Pull request by asottile-sentry**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@ PIP := python -m pip --disable-pip-version-check
 WEBPACK := yarn build-acceptance
 
 freeze-requirements:
-	@python -S -m tools.freeze_requirements sentry
+	@python3 -S -m tools.freeze_requirements sentry
```
Comment (joshuarli) [inline Makefile:5]: Just curious, why is this preferable?

**Review by joshuarli — APPROVED** [review]
I see the Slack discussion. Would be cool if we can eventually identify and mark a subset of tests that work with xdist and run them in their own matrix job.

File: Makefile (hunk 1 above)
Comment (asottile-sentry) [reply Makefile:5]: heh

```console
$ python
bash: python: command not found
```

Comment (joshuarli) [reply Makefile:5]: lol

**Pull request merged**